### PR TITLE
[cdc] Fixed constant value of org.apache.kafka prefix being shaded.

### PR DESCRIPTION
**Pull request by zhuangchong**
### Purpose

After the MySqlRecordParser.class in paimon-flink-1.18-0.6.0-incubating.jar is decompiled, the string `org.apache.kafka.connect.data.Decimal` is shaded to `org.apache.flink.kafka.shaded .org.apache.kafka.connect.data.Decimal`

image

After this PR:

MySqlRecordParser.class decompilation:
image

DebeziumSchemaUtils.class decompilation:
image

KafkaSinkFunction.class decompilation:
image

### CODE REVIEW

**Comment by JingsongLi** [comment]
+1

File: paimon-flink/paimon-flink-cdc/pom.xml [hunk 1]
```diff
@@ -334,6 +334,10 @@ under the License.
                                 <relocation>
                                     <pattern>org.apache.kafka</pattern>
                                     <shadedPattern>org.apache.flink.kafka.shaded.org.apache.kafka</shadedPattern>
+                                    <!-- TODO Constants will be shaded. see https://issues.apache.org/jira/browse/MSHADE-156 -->
+                                    <excludes>
+                                        <exclude>org.apache.kafka.connect.data.Decimal</exclude>
+                                    </excludes>
```
Comment (yuzelin) [inline paimon-flink/paimon-flink-cdc/pom.xml:340]: Please don't exclude this.

Comment (zhuangchong) [reply paimon-flink/paimon-flink-cdc/pom.xml]: done.

**Review by yuzelin — APPROVED** [review]
+1

**Pull request merged**